django/contrib/admin/widgets.py

```diff
@@ -145,13 +145,13 @@ class ForeignKeyRawIdWidget(forms.TextInput):
     in a <select> box.
     """
     def __init__(self, rel, admin_site, attrs=None, using=None):
-        self.rel = rel
```

Comment:
This is a mistake caused by global search-and-replace. There is no need to rename widget.rel. I'll fix this one.